plan/statistics/statistics_test.go

```diff
@@ -16,7 +16,7 @@ package statistics
 import (
 	"testing"
 
-	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/proto"
```

Comment:
gogo proto is used to generate .pb file, which is not necessary to be used in TiDB.